lib/jsonapi.rb

```diff
@@ -0,0 +1,15 @@
+module JSONAPI
+  require_relative 'jsonapi/rails'
```

Comment:
Yeah but here this is the whole path. Plus it's not like we require shittons of stuff so I'd be in favor of using `require` everywhere.